moninag/static/src/ts/auth/auth.module.ts

```diff
@@ -4,20 +4,23 @@ import { FormsModule } from '@angular/forms';
 import { HttpModule, BaseRequestOptions } from '@angular/http';
 import { RouterModule } from '@angular/router';
 import { MaterialModule } from '@angular/material';
-
-import { AlertService, AuthenticationService, UserService } from './_services/index';
+import { FlexLayoutModule } from "@angular/flex-layout";
+import { AlertService, AuthenticationService, UserService, ResetPasswordService } from './_services/index';
 import { AuthComponent } from './auth.component'
 import { MessageComponent } from './message/message.component'
 import { LoginComponent } from './login/login.component'
 import { RegisterComponent } from './register/register.component'
-
+import { ResetPasswordComponent } from './reset_password/reset_password.component'
+import { ConfirmPasswordResetComponent } from './reset_password/confirm_password_reset.component'
+import { EqualValidator } from './reset_password/equal-validator.directive';
```

Comment:
It is good to separate built-in imports from ours with blank line.

It will be also easier to read/search imports if they sorted :slightly_smiling_face: